chassis/core/src/main/java/com/griddynamics/jagger/engine/e1/aggregator/workload/DurationLogProcessor.java

```diff
@@ -255,12 +248,14 @@ public StatisticsGenerator generate() throws IOException {
             persistAggregatedMetricValue(Math.rint(globalStatisticsCalc.getStandardDeviation()) / 1000D, latencyStdDevDesc);
 
             Long startTime = aggregationInfo.getMinTime();
-            Long endTime = aggregationInfo.getMaxTime();
```

Comment:
can it be an issue of com.griddynamics.jagger.storage.fs.logging.ChronologyLogAggregator?

you are fixing it here, but looks like aggregation info max time is set incorrectly in the ChronologyLogAggregator